.github/workflows/main.yaml

```diff
@@ -36,11 +36,16 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18.x
-          cache: 'yarn'
-          cache-dependency-path: ./frontend/yarn.lock
```

Comment:
No longer necessary as cache action will pick up the shared `yarn.lock` from the root by default.